packages/account/src/Sections/Security/Passwords/sent-email-modal.jsx

```diff
@@ -81,28 +82,35 @@ const SentEmailModal = ({ identifier_title, is_open, is_unlink_modal, onClose, o
                     </Modal.Footer>
                 </React.Fragment>
             ) : (
-                <Modal.Body>
-                    <div onClick={onClose} className='sent-email__modal-close'>
-                        <Icon icon='IcCross' />
-                    </div>
-                    <SendEmailTemplate
-                        className='sent-email'
-                        title={localize('We’ve sent you an email')}
-                        subtitle={getSubtitle()}
-                        lbl_no_receive={localize("Didn't receive the email?")}
-                        txt_resend={localize('Resend email')}
-                        txt_resend_in={localize('Resend email in {{seconds}}s', { seconds: '{{seconds}}' })}
-                    >
-                        {getNoEmailContentStrings().map((item, idx) => (
-                            <div className='sent-email__content' key={idx}>
-                                <Icon icon={item.icon} size={32} />
-                                <Text size='xxs' as='p'>
-                                    {item.content}
-                                </Text>
-                            </div>
-                        ))}
-                    </SendEmailTemplate>
-                </Modal.Body>
+                <Div100vhContainer
+                    className='account__scrollbars_container-wrapper'
+                    is_disabled={isDesktop()}
+                    height_offset='80px'
+                >
+                    <Modal.Body>
+                        <div onClick={onClose} className='sent-email__modal-close'>
+                            <Icon icon='IcCross' />
+                        </div>
+                        <SendEmailTemplate
+                            className='sent-email'
+                            title={localize('We’ve sent you an email')}
+                            subtitle={getSubtitle()}
+                            lbl_no_receive={localize("Didn't receive the email?")}
+                            txt_resend={localize('Resend email')}
+                            txt_resend_in={localize('Resend email in {{seconds}}s', { seconds: '{{seconds}}' })}
+                            onClickSendEmail={onClickSendEmail}
```

Comment:
`onClickSendEmail` is the same as `onConfirm` can we re-use that?

Reply:
yes sure